src/fm/utils.cljc

```diff
@@ -230,7 +230,7 @@
                                 :args ~args-syms}))
 
          (if (args-anomaly?* ~args-syms)
-           (anom# ~args-syms)
+           (handler# ~args-syms)
```

Comment:
Oh man, this makes me much happier to see it called this way. Makes a lot more sense.